pkg/appruntime/chain/llmchain.go

```diff
@@ -55,7 +56,20 @@ func (l *LLMChain) Init(ctx context.Context, cli client.Client, args map[string]
 	return nil
 }
 
-func (l *LLMChain) Run(ctx context.Context, _ client.Client, args map[string]any) (outArgs map[string]any, err error) {
+func (l *LLMChain) Run(ctx context.Context, cli client.Client, args map[string]any) (outArgs map[string]any, err error) {
+	if args[base.ConversationKnowledgeBaseInArg] != nil {
+		chain := NewRetrievalQAChain(l.BaseNode)
```

Comment:
I do think we can combine `LLMChain` and `RetrievalQAChain` which makes `LLMChain` is a special(No knowledgebase required) conversation chain